server/controllers/admin/suppliers.js

```diff
@@ -66,26 +78,50 @@ function detail(req, res, next) {
   .done();
 }
 
-// POST /supplier
+/**
+ * @method create
+ *
+ * @description
+ * POST /supplier
+ *
+ * This method creates a new supplier entity in the database and sets up the
+ * creditor for the it.
+ */
 function create(req, res, next) {
   const data = db.convert(req.body, ['creditor_uuid']);
 
   // provide uuid if the client has not specified
-  data.uuid = db.bid(data.uuid || uuid());
+  const uid = data.uuid || uuid();
```

Comment:
The `uuid()` method of calling a version 4 uuid [is deprecated](https://github.com/broofa/node-uuid#deprecated-apis).  We should stick to `uuid,.v4()`.  I'm not sure how this got through the initial review, but I"ll update it!